### ISSUE TITLE
[bug]: Sending Image from txt2img to inpainting causes spinning circle and image doesn't load

### ISSUE THREAD

**Issue by mickr777** [invoke-ai/InvokeAI#1297]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cuda

### VRAM

12gb

### What happened?

Sending Image from txt2img/img2img to inpainting causes spinning circle and image doesn't load

no errors in cli or in web ui console

works fine from image gallery while under inpainting option

### Screenshots

![image](https://user-images.githubusercontent.com/115216705/198855835-7069befe-faae-417a-bbb5-f9ff551f95f1.png)

### Additional context

Same issue On Chrome on windows/android, Microsoft edge on windows and Firefox on linux

**Comment by Fortyseven** [comment]
Been seeing this, too, randomly. Didn't observe a pattern yet.

**Comment by Fortyseven** [comment]
Actually, I'm an idiot, I have a similar bug:

https://user-images.githubusercontent.com/156409/198863182-bd74080d-80af-44b5-b90b-10a6eea33da4.mp4

**Comment by Fortyseven** [comment]
Related, after trying inpainting from the clipboard again (same session) -- "images do not match":
```
>> Image generation requested: {'prompt': 'water geyser', 'iterations': 1, 'steps': 50, 'cfg_scale': 7.5, 'threshold': 0, 'perlin': 0, 'height': 512, 'width': 512, 'sampler_name': 'k_lms', 'seed': 619440448, 'progress_images': False, 'init_img': 'outputs/init-images/image.43902f49.png', 'strength': 0.75, 'fit': False, 'init_mask': 'iVBORw0KGgoAAAANSUhE...', 'bounding_box': {'x': 0, 'y': 0, 'width': 583, 'height': 1140}, 'variation_amount': 0}
ESRGAN parameters: False
Facetool parameters: False
>> Parsed prompt to FlattenedPrompt:[Fragment:'water geyser'@1.0]
>> using provided input image of size 583x1140
>> This input is larger than your defaults. If you run out of memory, please use a smaller image.
>> using provided input image of size 583x1140
>> Using recommended DDIM sampler for inpainting.
>> target t_enc is 37 steps
Generating:   0%|                                                                     | 0/1 [00:00<?, ?it/s]>> Running DDIMSampler sampling starting at step 13 of 50 (37 new sampling steps)
Decoding image: 100%|███████████████████████████████████████████████████████| 37/37 [00:07<00:00,  5.15it/s]
Generating:   0%|                                                                     | 0/1 [00:07<?, ?it/s]
images do not match


Traceback (most recent call last):
  File "/home/fortyseven/opt/ai/invoke-ai/backend/invoke_ai_web_server.py", line 796, in generate_images
    self.generate.prompt2image(
  File "/home/fortyseven/opt/ai/invoke-ai/ldm/generate.py", line 455, in prompt2image
    results = generator.generate(
  File "/home/fortyseven/opt/ai/invoke-ai/ldm/invoke/generator/base.py", line 98, in generate
    image_callback(image, seed, first_seed=first_seed)
  File "/home/fortyseven/opt/ai/invoke-ai/backend/invoke_ai_web_server.py", line 742, in image_done
    image = paste_image_into_bounding_box(
  File "/home/fortyseven/opt/ai/invoke-ai/backend/invoke_ai_web_server.py", line 1227, in paste_image_into_bounding_box
    im.paste(donor_image, bounds)
  File "/home/fortyseven/opt/ai/invoke-ai/venv/lib/python3.10/site-packages/PIL/Image.py", line 1630, in paste
    self.im.paste(im, box)
ValueError: images do not match
```

**Comment by psychedelicious** [comment]
Both issues I was able to reproduce reliably and both are fixed in #1301.

The former is related to the canvas registering itself for re-draws as internal state changes. It is possible there are other causes than switching tabs that haven't been found yet, but this specific cause is fixed.
